mm2src/mm2_core/Cargo.toml

```diff
@@ -8,6 +8,7 @@ doctest = false
 
 [features]
 new-db-arch = []
+track-ctx-pointer = []
```

Comment:
weird that this is only needed now. why?

Reply:
If you use a feature on a crate that isn't defined in its cargo manifest clippy will complain about it.